src/components/edit-project/editProject.js

```diff
@@ -75,15 +69,17 @@ angular.module("openshiftCommonUI")
                   cb(encodeURIComponent($scope.project.metadata.name));
                 }
 
-                showAlert({
-                  type: "success",
+                NotificationsService.addNotification({
+                  id: 'edit-project-success',
+                  type: 'success',
                   message: "Project \'"  + displayNameFilter(project) + "\' was successfully updated."
                 });
               }, function(result) {
                 $scope.disableInputs = false;
                 $scope.editableFields = editableFields($scope.project);
-                showAlert({
-                  type: "error",
+                NotificationsService.addNotification({
+                  id: 'edit-project-error',
```

Comment:
Decided just to remove the IDs. User could potentially try twice and not see the second notification.

Also updated notification messages and Logger'd failures.